orchagent/vnetorch.cpp

```diff
@@ -717,7 +717,7 @@ bool VNetBitmapObject::addIntf(const string& alias, const IpPrefix *prefix)
         intfMap_.emplace(alias, intfInfo);
     }
 
-    if (prefix)
+    if (prefix && gIntfsOrch->updateSyncdIntfPfx(alias, *prefix))
```

Comment:
I don't get this. If it is "vnet" interface, it will call the `VNetOrch::setIntf`. So how will this get added to `syncd `interfaces

Reply:
Right, ```intfsorch``` calls ```VNetOrch::setIntf -> VNetBitmapObject::addIntf```, and```addIntf``` creates VNET specific objects for the specific VNET interface but before that it also calls ```IntfsOrch::setIntf``` function which adds such interface to ```IntfsOrch::m_syncdIntfses```. So here interface is already in syncd map and we just add IP2ME prefix to the appropriate map in case it is not there.